### PR TITLE
Release System Charts for 2.5.10

### PR DESCRIPTION
**Pull request by thedadams**
https://github.com/rancher/rancher/issues/33528

### CODE REVIEW

File: charts/rancher-gke-operator/1.0.300/Chart.yaml
Comment (cbron) [inline charts/rancher-gke-operator/1.0.300/Chart.yaml]: This is already live ? And not supposed to be here according to: https://github.com/rancher/rancher/issues/33528
https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-gke-operator/1.0.300

File: charts/rancher-logging/0.2.5/Chart.yaml
Comment (cbron) [inline charts/rancher-logging/0.2.5/Chart.yaml:5]: This is also not in "new charts" issue: https://github.com/rancher/rancher/issues/33528
I believe we talked about this, but can someone track down.

File: charts/rancher-monitoring/v0.2.2/questions.yml
Comment (cbron) [inline charts/rancher-monitoring/v0.2.2/questions.yml]: curious that we are moving to patch for everything

**Review by cbron — CHANGES_REQUESTED** [review]

File: charts/rancher-gke-operator/1.0.300/Chart.yaml
Comment (thedadams) [reply charts/rancher-gke-operator/1.0.300/Chart.yaml]: I messaged you about this. This is live. The reason it is showing up is a little weird, but I am sure it has something to do with the fact that this PR for 2.5.9 came from `dev-v2.5.9` branch instead of `dev-v2.5` branch.

File: charts/rancher-logging/0.2.5/Chart.yaml
Comment (thedadams) [reply charts/rancher-logging/0.2.5/Chart.yaml:5]: Yes, @paynejacob reviewed and said these changes are good to go in here. I will update new charts issue.

**Review by paynejacob — DISMISSED** [review]

File: charts/rancher-gke-operator/1.0.300/Chart.yaml
Comment (cbron) [reply charts/rancher-gke-operator/1.0.300/Chart.yaml]: Can you try pulling in/rebasing release to this branch to see if it goes away.

Comment (thedadams) [reply charts/rancher-gke-operator/1.0.300/Chart.yaml]: I created a PR with this rebase: https://github.com/rancher/system-charts/pull/544

**Review by cbron — DISMISSED** [review]
lgtm. only problem is we literally have 3 different min versions: rc1, patch1, alpha1. Technically this should cause any problems, we didn't create any special 2.5 releases that I'm aware of, but we should standardize eventually. 
Will wait to merge until final sign off.

File: charts/rancher-eks-operator/1.0.800/questions.yaml
Comment (jiaqiluo) [inline charts/rancher-eks-operator/1.0.800/questions.yaml:2]: This is the only change outside those 3 new charts.

**Review by jiaqiluo — DISMISSED** [review]
LGTM

Files match the new chart in the issue. 
``` 
rancher-eks-operator | 1.0.1000  
rancher-gke-operator | 1.1.100  
rancher-logging | 0.2.5
```

**Review by aiyengar2 — COMMENTED** [review]
- [x] All charts in this PR are within https://github.com/rancher/rancher/issues/33528 and no additional charts are in this PR?

- rancher-eks-operator
  - [x] Latest version is `1.0.1000`?
  - [x] rancher_max_version of previous chart is `2.5.9`?
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
- rancher-gke-operator
  - [x] Latest version is `1.1.100`?
  - [ ] rancher_max_version of previous chart is `2.5.9`?
    - **Is this required? Currently, there is no rancher_max_version for the previous chart.**
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
    - **Inconsistent: currently is `2.5.10-patch1`**
- rancher-logging
  - [x] Latest version is `0.2.5`? 
  - [ ] rancher_max_version of previous chart is `2.5.9`?
    - **Is this required? Currently, there is no rancher_max_version for the previous chart.**
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
    - **Inconsistent: current version is `2.5.10-rc1` and the `questions.yaml` is named `questions.yml` instead.**

`rancher_min_version` and older `rancher_max_version` are inconsistent. IMO, we should standardize on having the latest chart have `rancher_min_version: 2.5.10-alpha1` and having the previous latest chart have `rancher_max_version: 2.5.9`, similar to what was established in https://github.com/rancher/system-charts/pull/527.

@cbron should we do a PR to fix this or leave it as is? Functionally, all charts are in a state where they will still work, but we should clean up / standardize IMO.

If it is decided that no changes are required, this PR LGTM.

**Review by paynejacob — APPROVED** [review]

File: charts/rancher-eks-operator/1.0.1000/questions.yaml
Comment (cbron) [inline charts/rancher-eks-operator/1.0.1000/questions.yaml:1]: I'm not sure questions.yaml actually works, as opposed to questions.yml. But as these are both kev2 operators it doesn't actually matter.

**Review by cbron — APPROVED** [review]
lgtm

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Pull request merged**
